database/db.js

```diff
@@ -2,16 +2,60 @@ var mongoose = require('mongoose');
 mongoose.connect(`mongodb://localhost/ARList`, {useNewUrlParser:true});
 // create schema
 const AReviewSchema = new mongoose.Schema({
-  reviewId: { type: Number, unique: true },
+  reviewId: Number,
   reviewRating: Number,
-  reviewDate: Date(),
+  reviewDate: String,
   reviewText: String,
   reviewerName: String
 })
 
 // create model
-const AReview = mongoose.model('Review', AReviewSchema);
+var AReview = mongoose.model('review', AReviewSchema);
 
+// random name generator
+var RandomNameGenerator = () => {
+  let firstName = ["Julia", "Rob", "Jenny", "Kai", "Jon"],
+    lastName = ["Gens", "Nolan", "Shamoo", "Dong", "Yang"]
+
+  return `${firstName[Math.floor(Math.random() * 5)]} ${lastName[Math.floor(Math.random() * 5)]}`;
+};
+
+// random date generator
+var RandomDateGenerator = () => {
+  let months = ['Jan','Feb','Apr','May','June','July','Aug','Sept','Oct','Nov','Dec'];
+
+  let randomMonth = months[Math.floor(Math.random() * 12)];
+  return `${randomMonth} ${Math.ceil(Math.random() * 31)}, ${Math.ceil(Math.random() * 2) + 2017}`;
+};
+
+
+// seeding function
+const save = (data => {
+  data.forEach(review => {
+    var newReview = new AReview ({
+      reviewId: review.reviewId,
+      reviewRating: Math.ceil(Math.random() * 5),
+      reviewDate: RandomDateGenerator(),
+      reviewText: review.reviewText,
+      reviewerName: RandomNameGenerator()
+    })
+    newReview.save();
+    console.log(newReview)
+  })
+});
+
+// database find
+const getReviews = (callback) => {
+  AReview.find({}, (err, results) => {
```

Comment:
Question here - are you always going to grab all the reviews from the database?  

I might have missed it somewhere... but will you be selecting a random number of reviews for each movie (ex: one movie has 15 reviews and one has 27 reviews)?  Or do you plan on implementing that on the client end (if that makes sense)? :)

Reply:
Haven't decided yet but it makes more sense to only grab the reviews as needed.  In that case, I'll need to assign movie titles to each review and revise the getReviews function to include movie name.

Thanks for catching that!